### ISSUE TITLE
BackgroundUpload: TypeError: 0, _$$_REQUIRE(_dependencyMap[4], "../../../..").uuidv4 is not a function (it is undefined)

### ISSUE THREAD

**Issue by GunnarAK** [numandev1/react-native-compressor#223]
First of all, thank you for this package it seems very promising certainly with the state of other background uploaders in the RN world!

### Current behavior
When attempting to perform a MULTIPART `backgroundUpload` it throws immediately the error:
```
TypeError: 0, _$$_REQUIRE(_dependencyMap[4], "../../../..").uuidv4 is not a function (it is undefined)
```

### Expected behavior
Upload should start.

### Platform

- [X] Android
- [X] iOS

### React Native Version
0.72.6

### React Native Compressor Version
1.8.11 till latest

### Reproducible Steps And Demo

1. install package `yarn add react-native-compressor`
2. Add following code
```
const upload = async ({
  abortSignal,
  file,
  fileIndex,
  onUploadProgress,
  uploadUrl,
}: RequestParams): Promise<UploadFileResponse> => {
  const fileUrl = isAndroid ? file.originalPath : file.uri;

  if (!fileUrl) throw new Error("Invalid file");

  const response = await backgroundUpload(
    uploadUrl,
    fileUrl,
    {
      uploadType: UploadType.MULTIPART,
      httpMethod: "POST",
      headers,
      fieldName: "file",
    },
    (written, total) => onUploadProgress(Math.round((written / total) * 100)),
  );

  return response;
};
```
3. Select image with an image picker such as `react-native-image-picker`.
4. Press your 'Upload' button
5. Error is thrown by `react-native-compressor`

**Comment by numandev1** [comment]
@GunnarAK it is working on my side, can you try change this line and check
https://github.com/numandev1/react-native-compressor/blob/e59879096698371c9e4a3e2197a485bd90406a87/src/utils/Uploader.tsx#L5

`import { uuidv4 } from './index';`

**Comment by GunnarAK** [comment]
Hi @numandev1 ! Yep that fixed it! 🚀 

I replaced it in these two files:
- `node_modules/react-native-compressor/lib/module/utils/Uploader.js`
- `node_modules/react-native-compressor/src/utils/Uploader.tsx`

**Comment by numandev1** [comment]
@GunnarAK fixed in `1.8.13` version

**Comment by GunnarAK** [comment]
Man that's fast! And it works like a charm ⭐
